pkgs/tools/misc/otel-cli/default.nix

```diff
@@ -1,24 +1,28 @@
-{ lib, bash, buildGoModule, fetchFromGitHub, getent, stdenv }:
+{ lib, bash, buildGoModule, fetchFromGitHub, getent, stdenv, nix-update-script }:
 
 buildGoModule rec {
   pname = "otel-cli";
-  version = "0.0.20";
+  version = "0.1.0";
 
   src = fetchFromGitHub {
     owner = "equinix-labs";
     repo = pname;
     rev = "v${version}";
-    hash = "sha256-bWdkuw0uEE75l9YCo2Dq1NpWXuMH61RQ6p7m65P1QCE=";
+    hash = "sha256-iYlyokBAS5KQUYq83zhKWH/Vulq7prQdceFpeBJN2PI=";
   };
 
-  vendorHash = "sha256-IJ2Gq5z1oNvcpWPh+BMs46VZMN1lHyE+M7kUinTSRr8=";
+  vendorHash = "sha256-5c5uDp5KVo/DYAM5F76ivtT52+lNBheVmjAjmq6EJFk=";
 
-  preCheck = ''
-    ln -s $GOPATH/bin/otel-cli .
-  '' + lib.optionalString (!stdenv.isDarwin) ''
-    substituteInPlace main_test.go \
-      --replace 'const minimumPath = `/bin:/usr/bin`' 'const minimumPath = `${lib.makeBinPath [ getent ]}`'
-  '';
+  preCheck =
+    ''
+      ln -s $GOPATH/bin/otel-cli .
+    ''
+    + lib.optionalString (!stdenv.isDarwin) ''
+      substituteInPlace main_test.go \
+        --replace 'const minimumPath = `/bin:/usr/bin`' 'const minimumPath = `${lib.makeBinPath [getent]}`'
+    '';
```

Comment:
Also for this, if there is no actual change.

Reply:
this is probably their linter fixing up stuff that nixpkgs does not care about

Reply:
alejandra strikes again! apologies yall, had to do some hunk resets and `:noa w`

Reply:
yeah, I had to turn it off for this repo, hopefully the rfc will close soon